cpg_pipes/config-template.toml

```diff
@@ -0,0 +1,254 @@
+[workflow]
+# Access level (test, standard, full). Translates into the storage and inputs 
+# namespace (test or main):
+access_level = 'standard'
+
+# Dataset name to write intermediate files and use as hail billing project,
+# (unless specified in hail.hail_billing_project):
+#dataset = 'seqr'
+
+# Datasets to load inputs. If not provided, datasets will be determined 
+# automatically based on the input provider implementation:
+#datasets = []
+
+# Datasets to skip:
+#skip_datasets = []
+
+# Samples to skip:
+skip_samples = [
```

Comment:
Should this be commented out in the template?

Reply:
Thanks, fixed!